docs:  dotnet new in getting started with Toolkit (#978)
unoplatform/uno.toolkit.ui: doc/getting-started.md

- [Getting Started with Material for Toolkit](./material-getting-started.md)
- [Getting Started with Cupertino for Toolkit](./cupertino-getting-started.md)

## Installation

> I created the  project using 

Reviewer: ```suggestion
The Uno Toolkit library can be included in your application by using the [Visual Studio Wizard](xref:Uno.GettingStarted.CreateAnApp.VS2022) and/or the [`dotnet new` templates](xref:Uno.GetStarted.dotnet-new)
```

        </ResourceDictionary.MergedDictionaries>
    </ResourceDictionary>
    ```
### [**dotnet new**](#tab/dotnet-new)

> 1. When creating a new project using [dotnet new template](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=windows#uno-platform-application) ensure that you include `-toolkit` in dotnet new  command . For example  `dotnet new unoapp -preset blank -o MyApp -toolkit`. Including the `-toolkit` option in the command ensures proper configuration of the Uno Toolkit for the newly created application.

Reviewer: ```suggestion
1. When creating a new project using the `dotnet new` templates, make sure to include the [`-toolkit` option](xref:Uno.GettingStarted.UsingWizard.Features#toolkit) in the `dotnet new unoapp` command.

`dotnet new unoapp -preset blank -o MyApp -toolkit` 
```

> 2. To incorporate the Uno Toolkit into an existing project, navigate to the **App Code Library** in the Terminal. Once in the command line interface, execute the following command:`dotnet add package Uno.Toolkit.WinUI --version [latest version number]`. Ensure to replace `[latest version number] `with the actual latest version number, which can be obtained from [Uno.Toolkit.UI](https://www.nuget.org/packages/Uno.Toolkit.UI). This command adds the Uno Toolkit package to your project.

Reviewer: ```suggestion
2. To incorporate the Uno Toolkit into an existing application, you must add the [Uno.Toolkit.WinUI NuGet package](https://www.nuget.org/packages/Uno.Toolkit.WinUI) to the **App Code Library** project as well as the platform heads. An easy way to do this is through the command line. Run the following command within the directory of each `.csproj`:

`dotnet add package Uno.Toolkit.WinUI --version [latest version number]`. 

Make sure to replace `[latest version number] `with the actual latest NuGet version number.
```

        </ResourceDictionary.MergedDictionaries>
    </ResourceDictionary>
    ```

> ### [**dotnet new**](#tab/dotnet-new)

Reviewer 2: you shouldn't use numbered items here
it is not step1 followed by step2, but rather option1 vs option2

Reviewer 2: and, for "2." you need to included the part that adds the \<ToolkitResources> (step.4 from the Wizard section)

    
    `dotnet new unoapp -preset blank -o MyApp -toolkit` 

Option 2:- To incorporate the Uno Toolkit into an existing application, you must add the [Uno.Toolkit.WinUI NuGet package](https://www.nuget.org/packages/Uno.Toolkit.WinUI) to the **App Code Library** project as well as the platform heads. An easy way to do this is through the command line. Run the following command within the directory of each `.csproj`:       
    
    `dotnet add package Uno.Toolkit.WinUI --version [latest version number]. 

 Make sure to replace `[latest version number] `with the actual latest NuGet version number.
 
 Add the resources to `AppResources.xaml`:

    ```xml
    <ResourceDictionary>
        <ResourceDictionary.MergedDictionaries>

            <!-- Load Uno.Toolkit.UI resources -->
            <ToolkitResources xmlns="using:Uno.Toolkit.UI" />

            <!-- Load custom application resources -->
            <!-- ... -->

        </ResourceDictionary.MergedDictionaries>
    </ResourceDictionary>

>     ```

Reviewer 2: ![image](https://github.com/unoplatform/uno.toolkit.ui/assets/2359550/213ae206-000a-4a8a-987a-6ba5114145d6)

pick a side:
- either make everything on indentation level 1 and on code level 2
- or keep `option x:` on lv1, each step on lv2, and code on lv3

Reviewer 2: also, notice how the xml code snippet is not picking up the syntax, showing the "\`\`\`xml" part?
you either want to indent by a level or use \`\`\`, not both

***

> #### Previous Installation Method

Reviewer 2: i would argue that "previous installation method" on the same heading level as "installation"
if you have chosen heading 2 `##` for installation, then do the same for other.
```suggestion
## Previous Installation Method
```

    </Application>
    ```

    

> 

Reviewer: Remove whitespace
